roles/clevis/tasks/create-tangcrypt.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Remove dropbear ssh initramfs packages
+  become: true
+  ansible.builtin.apt:
+    name: "{{ dropbear_packages }}"
+    state: absent
+    purge: true
+  notify: Clean-initramfs
+
+- name: "Disk {{ crypt_root_disk }} already bind to tang"
+  become: true
+  ansible.builtin.command: "clevis luks list -d {{ crypt_root_disk }} -s 2"
+  register: luksbind
+  ignore_errors: true
+  changed_when: false
+
+- name: Get adv handshake from tang server
+  become: true
+  ansible.builtin.uri:
+    url: "http://{{ tang_server_ip }}:{{ tang_server_port }}/adv"
```

Comment:
Introduce a new parameter tang_server_url as well as tang_server_protocol to build this URL.